third_party/ngen/ngen_compiler_fix.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright 2019-2024 Intel Corporation
+* Copyright 2020-2025 Intel Corporation
```

Comment:
Shouldn't the copyrights retain the starting year?

Reply:
Thanks, updated to the earliest year.